solr/solr-ref-guide/src/securing-solr.adoc

```diff
@@ -97,4 +97,4 @@ It is also strongly recommended that Solr listen to only those network interface
 
 == Enable Security Manager
 
-Solr can run in a Java Security Manager sandbox by setting `SOLR_SECURITY_MANAGER_ENABLED=true` via environment variable or in `solr.in.sh`/`solr.in.cmd`. This feature is incompatible with Hadoop.
+Solr runs in a Java Security Manager sandbox by default. Since this feature is incompatible with Hadoop, you can disable it by setting `SOLR_SECURITY_MANAGER_ENABLED=false` via environment variable or in `solr.in.sh`/`solr.in.cmd` if you are running Solr with Hadoop.
```

Comment:
I think we want to remove this whole section from this documentation page. If we fix the default here, then it is one less step the user has to worry about. Can we find a better place to document this hadoop stuff?

Reply:
I will take a look and add it to the PR. Since the default is true, I do agree that the Hadoop bit should be on its own. This comment makes me want to test to see if the security manager is still incompatible with Hadoop. I don't know enough of the history to know if it should or should not work.

Reply:
I need to conduct a variety of tests before writing a definitive statement about its functionality anywhere in the docs to ensure the information is accurate. Even if I experience some success, I will still include that it may not work based on the previous docs. I will also maybe consult one of the committers  involved in HDFS or (you included) for guidance on what to test or if it will not work for sure for reasons beyond my scope of knowledge at this time.

Reply:
I think we could something about the Solr Security Manager here: https://github.com/apache/lucene-solr/blob/master/solr/solr-ref-guide/src/hadoop-authentication-plugin.adoc